src/ParallelAlgorithms/LinearSolver/Multigrid/Tags.hpp

```diff
@@ -0,0 +1,205 @@
+// Distributed under the MIT License.
+// See LICENSE.txt for details.
+
+#pragma once
+
+#include <array>
+#include <cstddef>
+#include <optional>
+#include <vector>
+#include <unordered_set>
+
+#include "DataStructures/DataBox/PrefixHelpers.hpp"
+#include "DataStructures/DataBox/Tag.hpp"
+#include "DataStructures/Matrix.hpp"
+#include "DataStructures/Variables.hpp"
+#include "Domain/Tags.hpp"
+#include "NumericalAlgorithms/Spectral/Mesh.hpp"
+#include "Options/Auto.hpp"
+#include "Options/Options.hpp"
+#include "Parallel/PupStlCpp17.hpp"
+#include "Utilities/TMPL.hpp"
+
+namespace LinearSolver::multigrid {
+
+namespace OptionTags {
```

Comment:
Should this be documented? I'm not sure we have a policy for option tags namespaces or the tags themselves

Reply:
Barely any option tags namespace or option tags themselves are currently documented. We should figure out a good way to do that, but this PR is probably not the place for that :smile: (The thing with option tags is that the docs should really be in their help string)